edx/edx-exams
# [MST-1473] Proctoring Service Launch

**JIRA:** [MST-1473](https://2u-internal.atlassian.net/browse/MST-1473)

**Description:** 

This pull request uses the Proctoring Service implementation in a [branch](https://github.com/openedx/xblock-lti-consumer/pull/275) of `xblock-lti-consumer` to demonstrate a proctoring launch using LTI.

**Author concerns:**

The bulk of the code is in the associated [pull request](https://github.com/openedx/xblock-lti-consumer/pull/275). Most of my concerns are in that repository.

**Dependencies:**
* https://github.com/openedx/xblock-lti-consumer/pull/254
* https://github.com/openedx/xblock-lti-consumer/pull/275

**Installation instructions:**

Please see "Testing Instructions" in this [pull request](https://github.com/openedx/xblock-lti-consumer/pull/275).

**Testing instructions:**

Please see "Testing Instructions" in this [pull request](https://github.com/openedx/xblock-lti-consumer/pull/275).

**Merge checklist:**

**Post merge:**
